recipes/pyopcode/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+BUILD_CONFIG=Release
+
```

Comment:
Could we try using the `toolchain`? Please see this PR ( https://github.com/conda-forge/thrift-cpp-feedstock/pull/4 ) for and example of how to do this.